docs/source/running_bots.rst

```diff
@@ -0,0 +1,109 @@
+Running bots as participants
+============================
+
+Dallinger supports writing bots using the Selenium framework that participate in
```

Comment:
=> "Dallinger supports using the Selenium framework to write bots that participate in experiments."